ckanext/ontario_theme/templates/home/help.html

```diff
@@ -10,64 +10,226 @@
-      
-      <h2 id="how-to-archive-data">How to archive data</h2>
-      <p>There is no need to archive data. Instead, continually add new data when it is collected and making it clear to others what time series they are viewing.</p>
-      <p>When you have data that is in a time series (for example, collected annually or reflecting an annual cycle of work), ensure that your file names include the time series if the data will be collected in separate files. You can also collect all time series in the same file and update the file, as needed.</p>
+  
+      <h2><a name="i-cant-find-the-dataset-im-looking-for"></a>Q. I cannot find the dataset I’m looking for.</h2>
+      <div>
+        <p>
+          A. Try using the <a href="/dataset" target="_blank">catalogue search page</a>. Enter your keywords in the search bar and refine your search, using the filters provided.
+        </p>
+        <p>
+          If you still cannot find the dataset you want, there may be a couple of reasons why.
+        </p>
+        <p>
+          The dataset you want may:
+        </p>
+        <ul>
+          <li>
+            not listed in the beta catalogue, but can be found in the <a href="http://www.ontario.ca/data" target="_blank">current catalogue</a>. While the beta catalogue is in its testing phase, it will have some, but not all, of the datasets from the data catalogue.
+            <ul>
+              <li>
+                <a href="http://www.ontario.ca/data">Search the current data catalogue</a>
+              </li>
+              <li>
+                <a href="/about#beta">Learn more about the beta stage</a>
+              </li>
+            </ul>
+          </li>
+          <li>
+            not listed in the beta catalogue or in the current catalogue
+            <ul>
+              <li>
+                Email us at <a href="mailto:opengov@ontario.ca">opengov@ontario.ca</a> or <a href="https://www.ontario.ca/feedback/contact-us?id=14655&nid=74524">use our contact form</a> to tell us what data you are looking for. We will review your request and reply. 
+              </li>
+           </ul>
+          </li>
+        </ul>
+      </div>
+
+
+      <h2><a name="how-do-i-download-datasets"></a>Q. How do I download datasets?</h2>
+      <div>
+        <p>
+          A. On each page in the catalogue, where data is available, you will find the data icon under the heading <strong>Data and Resources</strong>. 
+        </p>
+        <p>
+          To the right of the data icon, you will find a either a <strong>Download</strong> or <strong>Open</strong> button.
+        </p>
+        <figure>
+          <img src="/images/help/Dataset Page - Download Button.jpg" alt="A screenshot of the data and resources section of the dataset description page. Each file in the dataset is listed. To the right of each file is a preview button and a download button. An arrow is overlaid onto the screeshot that points to the download button." />
```

Comment:
Ideally these file names for the images would be lowercase with no spaces. `kebab-case` would be good to align with other system generated urls.